### PR TITLE
General code improvement part 2

### PR DESCRIPTION
**Pull request by pzhanggit**
Missed a few places in the previous PR #53 of general code improvement:
improve data normalization and add restart

### CODE REVIEW

File: run_config_input.py [hunk 1]
```diff
@@ -384,6 +384,21 @@ def run_normal_config_file(config_file="./examples/configuration.json"):
     with open("./logs/" + model_with_config_name + "/config.json", "w") as f:
         json.dump(config, f)
 
+    if (
+        "continue" in config["NeuralNetwork"]["Training"]
```
Comment (streeve) [inline run_config_input.py:388]: Is this our only optional argument? Just curious

File: data_utils/raw_dataset_loader.py [hunk 2]
```diff
@@ -101,7 +111,7 @@ def __transform_input_to_data_object_base(
             for icomp in range(graph_feature_dim[item]):
                 it_comp = graph_feature_col[item] + icomp
                 g_feature.append(float(graph_feat[it_comp].strip()))
-        data_object.y = tensor([g_feature])
```
Comment (streeve) [inline data_utils/raw_dataset_loader.py:104]: Is this a bug? Or just cleaner this way?

**Review by streeve — APPROVED** [review]
Looks good. I'm wary of merging too much before we get the improved tests in, but I wouldn't block this

File: run_config_input.py (hunk 1 above)
Comment (pzhanggit) [reply run_config_input.py:388]: > Is this our only optional argument? Just curious

No, we have at least one more, i.e., "denormalize_output". By default it is "False".

File: data_utils/raw_dataset_loader.py (hunk 2 above)
Comment (pzhanggit) [reply data_utils/raw_dataset_loader.py:104]: > Is this a bug? Or just cleaner this way?

Yeah, it is a typo.

**Pull request merged**